sys/conf/kern.mk

```diff
@@ -3,7 +3,7 @@
 #
 # Warning flags for compiling the kernel and components of the kernel:
 #
-CWARNFLAGS?=	-Wall -Wredundant-decls -Wnested-externs -Wstrict-prototypes \
+CWARNFLAGS=	-Wall -Wredundant-decls -Wnested-externs -Wstrict-prototypes \
```

Comment:
Is this needed?

Reply:
I think so, this was added in bb8961f6f257c2ce1c4 and I think bsd.cpu.mk is still changing the CWARNFLAGS when in purecap.

Reply:
Ok, this probably isn't upstreamable, but it's likely fine for now.